packages/shared/src/components/tooltips/ChangelogTooltip.tsx

```diff
@@ -62,17 +63,14 @@ function ChangelogTooltip<TRef extends HTMLElement>({
         return;
       }
 
-      const sendMessage = globalThis?.browser?.runtime?.sendMessage;
```

Comment:
Great find on this. We never knew beforehand that the polyfill should be prioritized **all the time** on the extension package.